api/audit.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/netlify/gotrue/models"
+)
+
+var filterColumnMap = map[string][]string{
+	"author": []string{"actor_email", "actor_name"},
```

Comment:
Just to confirm I'm reading this right, there is only one query right now (author) ?

Reply:
Yes. It didn't sound like we had any others that would apply at the moment.

Reply:
👍